src/MIDebugEngine/ResourceStrings.resx

```diff
@@ -253,4 +253,7 @@ See https://aka.ms/miengine-gdb-troubleshooting for details.</value>
   <data name="DebuggingWillAbort" xml:space="preserve">
     <value>Debugging will now abort.</value>
   </data>
+  <data name="Warning_UncheckedExceptionsInCheckedCategory" xml:space="preserve">
+    <value>Warning: One or more C++ exceptions are unchecked while '&lt;All C++ Exceptions not in this list&gt;' is enabled. This is unsupported for GDB and the debugger will break on all C++ exceptions.</value>
```

Comment:
I am fairly certain you need a new line at the end of this (or to add it when creating the `OutputMessage` object).

Reply:
You are correct. I do need a new line. However, I tried adding `\n` to the end of the string in the .resx file, and it shows up in the Output window as well. 

```
Warning: One or more C++ exceptions are unchecked while '<All C++ Exceptions not in this list>;' is enabled. This is unsupported for GDB and the debugger will break on all C++ exceptions.\n
```

This seems to be a common problem when adding the new line character in resx files...

Reply:
Just open up the resource editor in Visual Studio and add a real new line (hit enter)

Reply:
In `.resx`, new lines are just `enter` instead of escaped new line character. 

E.g.
```xml
  <data name="Error_Message" xml:space="preserve">
    <value>First Line
Second Line
Third Line

Fifth Line</value>
```